service/build.gradle

```diff
@@ -16,7 +16,20 @@ plugins {
 apply plugin: 'org.springframework.boot'
 apply plugin: 'io.spring.dependency-management'
 apply plugin: 'info.solidsoft.pitest'
-apply plugin: 'org.owasp.dependencycheck'
+
+// https://jeremylong.github.io/DependencyCheck/dependency-check-gradle/configuration.html
+dependencyCheck {
+  // Specifies if the build should be failed if a CVSS score above a specified level is identified.
```

Comment:
this comment is obsolete

Reply:
Thanks, removed and cleaned up a bit so it shows less changes.